code/modules/antagonists/eldritch_cult/eldritch_effects.dm

```diff
@@ -222,28 +260,50 @@
 	anchored = TRUE
 	resistance_flags = FIRE_PROOF | UNACIDABLE | ACID_PROOF
 	invisibility = INVISIBILITY_OBSERVER
+	///We cannot use icon_state since this is invisible, functions the same way but with custom behaviour.
+	var/image_state = "reality_smash"
+	///Who can see us?
+	var/list/minds = list()
```

Comment:
I dislike this being tracked individually on every reality_smash. I would think all the reality_smashes don't hand the images themselves and instead the singleton reality_smash_manager or whatever handles that.

Reply:
Its pretty much what we tried to do before and it was bugged so yeah :/